src/Payments.sol

```diff
@@ -119,6 +134,26 @@ contract Payments is
         _;
     }
 
+    function terminateOperator(address operator) external {
```

Comment:
Hm. Looking at how this turned out.... I'm wondering if we want to just do this off-chain. I.e., make the client send one message per operator per token to unapprove and set the allowances to zero. Up to you, I'm just not sure if the extra complexity/state is worth it.

Reply:
Done.